model-server/src/test/kotlin/org/modelix/model/server/StoreClientTest.kt

```diff
@@ -143,4 +145,35 @@ abstract class StoreClientTest(val store: IStoreClient) {
 
         assertEquals(setOf<String?>(value3), valuesSeenByListener)
     }
+
+    @Test
+    fun nestedSuspendableTransactionAreOnlyDispatchedOnce() = runTest {
+        // Arrange
```

Comment:
Small nitpick: While I think it's great to use the Arrange-Act-Assert structure, I feel like adding these comments each time just produces noise. I'd prefer just visibly separating the parts with new lines.